docs/UserGuide.md

```diff
@@ -444,7 +444,8 @@ Examples:
 * `list` displays all students and lessons in TutorAid by only showing their names and list indexes.
 * `list -a` displays all students and lessons in TutorAid while showing all of their fields' data.
 
-### Adding a student to a lesson: `add -sl`
+
+### Adding student(s) to lesson(s): `add -sl`
 {:.no_toc}
 Tells TutorAid that a student attends a particular lesson.
 
```

Comment:
A few nitpicks on this:
* Need to update the command format to be `add -sl s/STUDENT_INDEX... l/LESSON_INDEX...` instead.
* The line `Adds the student at the specified student index to the lesson at the specified lesson index.` can be changed to `Adds the students at the specified student indexes to the lessons at the specified lesson indexes.`
* The example can perhaps be changed to "`add -sl s/1 2 3 l/2` adds the students with indexes 1, 2 and 3 into the lesson with index 2."

These issues apply to the `del -sl` command too